packages/create-experience-builder/src/ctflClient.ts

```diff
@@ -1,202 +1,135 @@
+import {
+  CREATE_CONTENT_ENTRY_REQUEST_BODY,
+  CREATE_CONTENT_TYPE_REQUEST_BODY,
+} from './constants.js';
 const baseUrl = process.env.BASE_URL || 'https://api.contentful.com';
-const contentLayoutType = 'DevLayout';
+const defaultExperienceName = `CLI Generated Experience Name`;
+const defaultExperienceNameId = `cliGeneratedExperienceName`;
 
 export class CtflClient {
-  public space: { name: string; id: string } | undefined;
-  private authToken: string | undefined;
-  private accessToken: string = '';
-  private previewAccessToken: string = '';
+  public space?: { name: string; id: string };
+  public org?: { name: string; id: string };
+  public apiKey?: {
+    accessToken: string;
+    name: string;
+    previewKeyId: string;
+    previewAccessToken?: string;
+  };
+
+  public previewEnvironment?: {
+    name: string;
+    id: string;
+    url: string;
+  };
+
+  private authToken?: string;
   private authTokenCreatedFromApi = false;
 
   async createApiKeys() {
     //Create API key
-    type ApiKeyReturn = { accessToken: string; preview_api_key: { sys: { id: string } } };
-    const apiKeys = await this.apiCall<ApiKeyReturn>(`/spaces/${this.space?.id}/api_keys`, {
-      body: '{"name":"Experience Builder Dev Keys"}',
+    type ApiKeyReturn = {
+      accessToken: string;
+      name: string;
+      preview_api_key: { sys: { id: string } };
+    };
+    const apiKey = await this.apiCall<ApiKeyReturn>(`/spaces/${this.space?.id}/api_keys`, {
+      body: '{"name":"Experience Builder Keys"}',
       method: 'POST',
     }).then((val) => {
       return {
         accessToken: val.accessToken,
+        name: val.name,
         previewKeyId: val.preview_api_key.sys.id,
       };
     });
-    this.accessToken = apiKeys.accessToken;
+    this.apiKey = apiKey;
+    return apiKey;
+  }
+
+  async getApiKeys() {
+    type ApiKeyReturn = {
+      items: { accessToken: string; preview_api_key: { sys: { id: string } }; name: string }[];
+    };
+    const apiKeys = await this.apiCall<ApiKeyReturn>(`/spaces/${this.space?.id}/api_keys`, {
+      method: 'GET',
+    }).then((val) => {
+      return val.items.map((item) => {
+        return {
+          accessToken: item.accessToken,
+          previewKeyId: item.preview_api_key.sys.id,
+          name: item.name,
+        };
+      });
+    });
+
+    return apiKeys;
+
+    // const apiKey = apiKeys.find((key) => key.name === 'Experience Builder Keys');
+
+    // if (!apiKey) {
+    //   return false;
+    // }
+
+    // // this.accessToken = apiKey;
+
+    // //Get Preview Key
+    // type PreviewKeyReturn = { accessToken: string };
+    // const previewKey = await this.apiCall<PreviewKeyReturn>(
+    //   `/spaces/${this.space?.id}/preview_api_keys/${apiKey.preview_api_key.sys.id}`,
+    //   {
+    //     method: 'GET',
+    //   }
+    // ).then((val) => {
+    //   return val.accessToken;
+    // });
+    // this.previewAccessToken = previewKey;
```

Comment:
Was this just some left over from testing? If there is really a need to keep this in the production code, I would add a comment with some reasoning to it.

Reply:
Agreed, I'll remove it unless @elylucas has any objections.

- [ ] remove or object

Reply:
Looks like it can be deleted to me